WordPressKit/Result+Callback.swift

```diff
@@ -8,6 +8,10 @@ extension Swift.Result {
         }
     }
 
+    func execute(completion: (Self) -> Void) {
+        completion(self)
+    }
```

Comment:
I used "execute", instead of "forward(to:)" suggested in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/721#discussion_r1482356171, to align with the existing `execute(onSuccess:onFailure:)` function.

Reply:
Fair 👍 

What do you think of removing the named parameter?

I feel the version without label reads better.

```swift
result.execute(completion: { print($0) })
result.execute({ print($0) })
```